eos_data_distribution/ndn/manifest.py

```diff
@@ -33,11 +33,13 @@
 # the manifest producer is an http Producer that answers on a different name
 class Producer(chunks.Producer):
     def __init__(self, name, url, session=None, *args, **kwargs):
-        self._getter = http.Getter(url, session)
-        self._getter.connect('data', lambda o, d: self._send_finish(d))
+        self._getter = http.Getter(url, onData=self._send_finish, session=session)
+        self._last_modified = http.get_last_modified(self._getter._headers)
+        if self._last_modified == None:
```

Comment:
Nitpick: I think `if not self._last_modified:` is more pythonic, but this works equally well.